TF_Settings_Web/src/Components/Pages/Camera/QuickSetup/Calibration/CalibrationCompleteScreen.tsx

```diff
@@ -47,7 +47,7 @@ const CalibrationCompleteScreen: React.FC<CalibrationCompleteProps> = ({ onLoad,
 
     const handleTFInput = (evt: CustomEvent<TouchFreeInputAction>): void => {
         if (evt.detail.InputType === InputType.MOVE || evt.detail.InputType === InputType.DOWN) {
-            setProgressToClick(evt.detail.ProgressToClick);
+            setProgressToClick(isHandPresent ? evt.detail.ProgressToClick : 0);
```

Comment:
I don't think that this is doing anything as it still requires an input action to clear the progress to click

Reply:
Good point 👍 will sort that now